packages/react/package.json

```diff
@@ -47,12 +46,11 @@
   },
   "homepage": "https://github.com/open-feature/js-sdk#readme",
   "peerDependencies": {
-    "@openfeature/core": "0.0.19",
```

Comment:
This is provided via the web-sdk, and specifying it here increased the likelyhood of peer conflicts.